application/src/main/java/com/sanction/thunder/resources/UserResource.java

```diff
@@ -63,21 +62,21 @@ public UserResource(PilotUsersDao usersDao, MetricRegistry metrics) {
   /**
    * Posts a new PilotUser to the database.
    *
-   * @param user The user to post to the database.
+   * @param pilotUser The user to post to the database.
    * @return The user that was created in the database.
    */
   @POST
-  public Response postUser(@Auth Key key, PilotUser user) {
+  public Response postUser(PilotUser pilotUser) {
```

Comment:
Any reason for changing the name from `user` to `pilotUser`?

Also, is there no auth on this endpoint anymore? We were going to have some sort of auth that was not user-based, right?